src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/SVAnnotate.java

```diff
@@ -0,0 +1,479 @@
+        String consequence = annotateDuplication(variantInterval, gtfTranscript);
+        if (MSVExonOverlapClassifications.contains(consequence)) {
+            return GATKSVVCFConstants.MSV_EXON_OVERLAP;
+        } else {
+            return consequence;  // TODO: MCNV classifications ???
+        }
+    }
+
+    private String annotateInversion(final SimpleInterval variantInterval, final GencodeGtfTranscriptFeature gtfTranscript) {
+        String consequence = GATKSVVCFConstants.INTRONIC;
+        final SimpleInterval transcriptInterval = new SimpleInterval(gtfTranscript);
+        if (variantSpansFeature(variantInterval, transcriptInterval)) {
+            consequence = GATKSVVCFConstants.INV_SPAN;
+        } else if (countBreakendsInsideFeature(variantInterval, transcriptInterval) == 1) {
+            consequence = GATKSVVCFConstants.LOF;
+        } else {
+            // both breakpoints inside transcript
+            final List<GencodeGtfFeature> gtfFeaturesForTranscript = gtfTranscript.getAllFeatures();
+            for (GencodeGtfFeature gtfFeature : gtfFeaturesForTranscript) {
+                final SimpleInterval featureInterval = new SimpleInterval(gtfFeature);
+                if (!variantOverlapsFeature(variantInterval, featureInterval)) {
+                    continue;
+                }
+                // TODO: if overlaps exon, it's LOF unless both breakpoints are in the same UTR?
+                if (gtfFeature.getFeatureType() == GencodeGtfFeature.FeatureType.EXON) {
+                    consequence = GATKSVVCFConstants.LOF;  // TODO: CDS or exon here?
+                } else if (gtfFeature.getFeatureType() == GencodeGtfFeature.FeatureType.UTR) {
+                    if (countBreakendsInsideFeature(variantInterval, featureInterval) == 2) {
+                        consequence = GATKSVVCFConstants.UTR;
+                    }
+                }
+            }
+        }
+        return consequence;
+    }
+
+    private String annotateTranslocation(final SimpleInterval variantInterval, final GencodeGtfTranscriptFeature gtfTranscript) {
+        // already checked for transcript overlap, and if a translocation breakpoint falls inside a gene it's automatically LOF
+        // TODO: eliminate unnecessary function or keep for aesthetics/future flexibility?
+        return GATKSVVCFConstants.LOF;
+    }
+
+    private String annotateBreakend(final SimpleInterval variantInterval, final GencodeGtfTranscriptFeature gtfTranscript) {
+        String consequence = annotateDeletionOrInsertion(variantInterval, gtfTranscript);
+        if (consequence.equals(GATKSVVCFConstants.LOF)) {
+            consequence = GATKSVVCFConstants.BREAKEND_EXON;
+        }
+        return consequence;
+    }
+
+    private void annotateTranscript(final SimpleInterval variantInterval, final StructuralVariantAnnotationType svType, final GencodeGtfTranscriptFeature transcript, final Map<String, Set<String>> variantConsequenceDict) {
```

Comment:
Some of these lines are awfully long.  The GATK coding standard says something about line lengths, but it's frequently ignored.  Nonetheless, it would be easier to read if you wrapped some of these super-long lines.